lua/statuscol.lua

```diff
@@ -171,7 +171,7 @@ local function get_lnum_action(minwid, clicks, button, mods)
   local cargs = callargs[args.mousepos.winid]
   if lnumfunc and cargs.sclnu then
     local placed = f.sign_getplaced(cargs.buf, {group = "*", lnum = args.mousepos.line})
-    if #placed[1].signs > 0 then
+    if placed and #placed[1].signs > 0 then
```

Comment:
Decided to fix also
![image](https://github.com/luukvbaal/statuscol.nvim/assets/33424304/63541f8f-86e3-4fc8-ae21-15dfff1774fe)